contracts/CDPRegistry.sol

```diff
@@ -103,7 +103,8 @@ contract CDPRegistry {
     function getCdpsByOwner(address owner) external view returns (CDP[] memory r) {
         address[] memory assets = cr.collaterals();
         CDP[] memory cdps = new CDP[](assets.length);
-        uint actualCdpsCount;
```

Comment:
I think missing initialization here might help save some gas, expecially in case the assets.length is 0 and it's not used/written to. So I don't think initialization here is necessary since uint type is anyway always initialized as 0